tests/integration/defs/examples/test_multimodal.py

```diff
@@ -478,9 +478,24 @@ def _test_llm_multimodal_general(llm_venv,
                 "If I had to write a haiku for this one"
             ])
 
-            print("Run mllama vision test...")
+            print("Run mllama vision test in with example image ...")
             _call_run_cmd(llm_venv, llm_root, run_cmd_vision, world_size)
 
+            print("multimodal_example_root: ", multimodal_example_root)
+            print("llm_root: ", llm_root)
+            run_cmd_vision = run_cmd.copy()
+            run_cmd_vision.extend([
+                "--cross_kv_cache_fraction=0.5",  # mllama uses cross attention
+                "--image_path",
+                os.path.join(
+                    llm_root,
+                    "tests/integration/test_input_files/dummy_image.jpg"),
```

Comment:
It is better to rename the image name since it is not a dummy image. Something like "excel_table_test" should be more clear.

Reply:
Thanks for your suggestion. Fixed.